.storybook/preview.js

```diff
@@ -1,7 +1,12 @@
 import { themes } from '@storybook/theming';
 
+import { initialize } from '../src/lib/locale.store';
+
 import '../src/app.css';
 
+// Initializes localized content globally for all stories
+initialize();
```

Comment:
This adds the true, fallback locale strings globally to Storybook. This means the components won't fallback to converting the key itself to the displayed string.